submissions/acerpo/index.html

```diff
@@ -11,56 +11,57 @@
+                    <label class="check headerLink" for="checkbox_toggle"><img class="nav__item_img" src="./img/popup-button.png" alt=""></label>
                     <div class="popup_content">
                         <div class="popup_content_body">
-                            <a class="popup_content_body_link" href="#"><img src="./img/calendar.png" alt="Calendar">Calendar</a>
-                            <a class="popup_content_body_link" href="#"><img src="./img/mail.png" alt="Mail">Mail</a>
-                            <a class="popup_content_body_link" href="#"><img src="./img/news.png" alt="News">News</a>
-                            <a class="popup_content_body_link" href="#"><img src="./img/maps.png" alt="Map">Map</a>
-                            <a class="popup_content_body_link" href="#"><img src="./img/files.png" alt="Files">Files</a>
-                            <a class="popup_content_body_link" href="#"><img src="./img/contacts.png" alt="Contacts">Contacts</a>
-                            <a class="popup_content_body_link" href="#"><img src="./img/tanslation.png" alt="Translate">Translate</a>
-                            <a class="popup_content_body_link" href="#"><img src="./img/videos.png" alt="Videos">Videos</a>
-                            <a class="popup_content_body_link" href="#"><img src="./img/search.png" alt="Search">Search</a>
-                            <a class="popup_content_body_link" href="#"><img src="./img/photos.png" alt="Photo">Photo</a>
-                            <a class="popup_content_body_link" href="#"><img src="./img/apps.png" alt="Apps">Apps</a>
-                            <a class="popup_content_body_link" href="#"><img src="./img/documents.png" alt="Document">Document</a>
+                            <ul class="manu-1">
+                                <li><a class="popup_content_body_link" href="#"><img src="./img/calendar.png" alt="Calendar">Calendar</a>
+                                <li><a class="popup_content_body_link" href="#"><img src="./img/mail.png" alt="Mail">Mail</a>
+                                <li><a class="popup_content_body_link" href="#"><img src="./img/news.png" alt="News">News</a>
+                                <li><a class="popup_content_body_link" href="#"><img src="./img/maps.png" alt="Map">Map</a>
+                                <li><a class="popup_content_body_link" href="#"><img src="./img/files.png" alt="Files">Files</a>
+                                <li><a class="popup_content_body_link" href="#"><img src="./img/contacts.png" alt="Contacts">Contacts</a>
+                                <li><a class="popup_content_body_link" href="#"><img src="./img/tanslation.png" alt="Translate">Translate</a>
+                                <li><a class="popup_content_body_link" href="#"><img src="./img/videos.png" alt="Videos">Videos</a>
```

Comment:
You forget about `alt`) [https://github.com/kottans/frontend-2022-homeworks/pull/26#discussion_r933176977](https://github.com/kottans/frontend-2022-homeworks/pull/26#discussion_r933176977)

Reply:
Looks like I misunderstood the article you posted.

Why is it indicated in the task that you must not forget to specify the data in ALT?

Reply:
We do not restrict your design, on previous courses students created a full-page search with a Google-like design. Also, if you add a logo you should fill `alt`.